src/medicalpage/SingleMedication.js

```diff
@@ -0,0 +1,150 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { EditStatusIcon } from './EditStatus';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { library } from '@fortawesome/fontawesome-svg-core';
+import {
+  faTrash,
+  faEdit,
+  faCheckSquare
+} from '@fortawesome/free-solid-svg-icons';
+import Input from '../components/Input';
+
+library.add(faTrash, faEdit, faCheckSquare);
+
+const StyledInput = styled.input`
```

Comment:
Didn't you have a component for this?

Reply:
Yes, I have, but the styling has to be more specific here.